### DISCUSSION TITLE
Android TV не меняют настройки.

### DISCUSSION THREAD

**Discussion by UserGit-2021** [dovecoteescapee/ByeDPIAndroid#145] [General]
Пытаюсь выбрать режим "Прокси" но закрывается выборы режима резко и остальные настройки тоже. Видимо баг с интерфейсом.

https://github.com/user-attachments/assets/d1016ab0-71c6-463b-97ed-0ce7eac0ed32

**Comment by kentoshka** [comment]
Такая ж херня на одном тв боксе
 / 
me too

**Comment by neitron10** [comment]
Этот баг решается очень просто. Необходимо вызвать на пульте тремя черточками верхнюю и нижнюю панель. Прсое чего кнопка исправно будет переключатся
